Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java

```diff
@@ -92,10 +100,7 @@ public void onClick(View v) {
   }
 
   private class GetPageList extends AsyncTask<String, Void, String> {
-    /*
-     * TODO(leesera): the constructor gets the DB as a parameter
-     */
-    public GetPageList() {
+    public GetPageList(DataBaseOpenHelper dataBaseOpenHelper) {
     }
```

Comment:
Can you explain what does this code mean? The code does not process the argument.